core/src/Cuda/Kokkos_Cuda_Instance.hpp

```diff
@@ -91,7 +91,7 @@ class CudaInternal {
   int m_cudaArch;
   unsigned m_multiProcCount;
   unsigned m_maxWarpCount;
-  unsigned m_maxBlock;
+  std::array<size_type, 3> m_maxBlock;
```

Comment:
Any particular reason to use `size_type` here?  It is indeed `unsigned` now but it could change in the future.
(Not necessarily asking you to change, just making sure you considered that)

Reply:
We were using `size_type` for the function returning this variable. Thus, it appeared that the intention was to use matching types. I see no other reason to use `unsigned int` otherwise since the Cuda function returns `int`.